### DISCUSSION TITLE
[Question]: What should I do after modifying the content in `data-provider/`?

### DISCUSSION THREAD

**Discussion by Kevin-free** [danny-avila/LibreChat#2801] [Q&A]
### What is your question?

When editing `api-endpoints.ts`, `data-service.ts`, `src/types.ts`, `src/react-query/react-query-service.ts` in the `data-provider/` file After coding, what needs to be done to update the project?

I tried executing `npm run build:data-provider`, and it took effect, but a strange problem occurred: all
`import xxx from 'librechat-data-provider';` are all marked with red prompts (although it does not affect the operation, it looks uncomfortable):

```Shell
Cannot find module 'librechat-data-provider' or its corresponding type declarations.
 There are types at 'xxx/node_modules/librechat-data-provider/dist/types/index.d.ts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext ', or 'bundler'.
```

Don't know what else to do? Or is it a problem with my version?

What’s even stranger is: after I restarted VSCode, all the red prompts disappeared.

Want your help or your practical steps?

### More Details

```shell
> node --version
v19.8.1
> npm --version
9.5.1
```

### What is the main subject of your question?

Other

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Please open questions like this in [discussion](https://github.com/danny-avila/LibreChat/discussions/)

This is a common issue when recompiling local packages. Use command palette in VS Code and Restart the typescript server. Since I do this a lot, I have this hotkeyed.
